mp.sh

```diff
@@ -36,32 +27,23 @@ if [ $# -gt 0 ]; then
   elif [ "$1" == "bundle" ]; then
     echo "Bundling spec into a single file..."
 
-    ${COMPOSE} run --rm bundler ash -c \
-    "mkdir -p /opt/dist && json-refs resolve schema.json -f > /opt/spec/dist/swagger.json"
+    ${COMPOSE} run --rm bundler ash -c "mkdir -p /opt/dist && json-refs resolve schema.json -f > /opt/spec/dist/swagger.json"
 
-    # On Linux fix permissions for the dist folder.
```

Comment:
Comments maken het juist wat overzichtelijker. Dan hoef je niet uit de code te halen wat er in een blok code gebeurt.